hack/create-testing-func-image.sh

```diff
@@ -4,12 +4,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KO_DOCKER_REPO="ttl.sh/$(head -c 128 </dev/urandom | LC_CTYPE=C tr -dc 'a-z0-9' | fold -w 8 | head -n 1)"
+KO_DOCKER_REPO="localhost:50000/knative/func"
 export KO_DOCKER_REPO
 
-REF_FILE=$(mktemp)
-
-ko build --image-refs "${REF_FILE}" --tags "30m" -B ./cmd/func
-
-yq -Y -i ".spec.steps[0].image = \"$(cat "${REF_FILE}")\"" \
```

Comment:
The thing is this patch does not work anymore. The task is not loaded from local FS but from PR branch so local modifications have no effect.